examples/models/deep_mnist/DeepMnist.py

```diff
@@ -1,18 +1,26 @@
 import tensorflow as tf
 import numpy as np
+import os
 
 class DeepMnist(object):
     def __init__(self):
+        self.loaded = False
```

Comment:
Should this be set explicitly by the user? The other alternative could be to handle this without the user knowing, and basically assume loaded is true if, for example "load" returns, or even returns True.